src/request_items.rs

```diff
@@ -141,7 +141,11 @@ impl FromStr for RequestItem {
     }
 }
 
-pub struct RequestItems(pub Vec<RequestItem>);
+#[derive(Default, Debug)]
+pub struct RequestItems {
+    pub items: Vec<RequestItem>,
+    pub request_type: RequestType,
```

Comment:
`request_type` is primarily about the request body, so maybe something like `body_type` or `request_body_type` would be more appropriate?